pkgs/by-name/ll/lldap-cli/package.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+
+, curl
+, jq
+, makeWrapper
+, lldap
+, util-linux
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: rec {
+  name = "lldap-cli";
```

Comment:
Prefer a combination of `pname` and `version`, instead of `name`. If a package is unversioned, the convention is to set the version to `0-unstable-YYYY-MM-DD`, where `YYYY-MM-DD` is where the date of the commit used. [See the section on package naming in `pkgs/README.md`.](https://github.com/NixOS/nixpkgs/blob/c10fb0df312867b0ae47a580818b317a9ae61f21/pkgs/README.md#package-naming)

In this case, `version` should be `0-unstable-2024-05-15`.